gosyntect.go

```diff
@@ -46,6 +46,8 @@ type Response struct {
 var (
 	// ErrInvalidTheme is returned when the Query.Theme is not a valid theme.
 	ErrInvalidTheme = errors.New("invalid theme")
+	// ErrInvalidRequest is returned when we get a 400 from syntect_server.
+	ErrInvalidRequest = errors.New("invalid request")
```

Comment:
I think we should make this:

```Go
ErrRequestTooLarge = errors.New("request too large")
```

Yes, it could also theoretically happen with other 400 responses but in practice the only 400 response we'll get will be in this case. The point of this package is to abstract away the obscurities of interacting with syntect_server over HTTP, so doing this makes sense

Reply:
Sure, if you're alright with making that assumption I'm fine with that.